src/threedidepth/calculate.py

```diff
@@ -434,6 +434,11 @@ def convert_using(self, calculator, band):
             band (int): Which band to write to.
         """
         no_data_value = self.no_data_value
+
+        if no_data_value is None:
+            # if no_data_value is not defined, assign NaN as the default
+            no_data_value = np.nan
```

Comment:
So you encountered a bathymetry without no data value in the wild...? You may consider defaulting to  `-9999.0` as `no_data_value` instead of NaN because that is more common in the 3Di world, but it is up to you.

Reply:
Yes, according to Leendert, he encountered one without a nodatavalue. 

@leendertvanwolfswinkel what do you think? Should the nodatavalue be default to `-9999.0` instead of `NaN`? I think it makes sense to default it to `-9999.0`

Reply:
both are fine to me, I think -9999.0 is slightly safer, because the algorithm expects a number